css-view-transitions-2/Overview.bs

```diff
@@ -888,20 +888,20 @@ and by applying ''view-transition-group'' to the internal element referencing th
 
 	<dl dfn-type=value dfn-for=view-transition-group>
 		: <dfn>normal</dfn>
-		:: The [=used value=] is the element's [=nearest containing group name=].
-
-		: <dfn>nearest</dfn>
-		:: The [=used value=] is the 'view-transition-name' [=computed value=] of the nearest ancestor whose 'view-transition-name' [=computed value=] is not ''view-transition-name/none''.
-
 		: <dfn>contain</dfn>
 		:: The [=used value=] is the element's [=nearest containing group name=].
 
-			Note: Descendant elements are contained within this group by default, as it would be their element's [=nearest containing group name=].
+			Note: An element with ''view-transition-group/contain'' becomes the [=nearest containing group name=] for its [=tree/descendants=].
+
+		: <dfn>nearest</dfn>
+		:: The [=used value=] is the 'view-transition-name' [=computed value=] of the nearest [=tree/ancestor=] whose 'view-transition-name' [=computed value=] is not ''view-transition-name/none''.
```

Comment:
Should we use the computed or used value of the name? With auto based naming seems like it should be used.

Reply:
Did an overhaul of this to be more algorithmic, PTAL